.cirrus.yml

```diff
@@ -55,6 +55,7 @@ env:
     DISTRO_NV:               # any {PRIOR_,}{FEDORA,DEBIAN}_NAME value
     VM_IMAGE_NAME:           # One of the "Google-cloud VM Images" (above)
     CTR_FQIN:                # One of the "Container FQIN's" (above)
+    CTR_DESIRED_DATABASE: boltdb # One of "", "sqlite", "boltdb"
```

Comment:
I'm still seeing icky CI names, and wonder if it's this? Did you mean to spell this as `CTR_`, or should it be `CI_`?

Reply:
This is probably my fault (typo in diff suggestion).